docs/CONTRIBUTING.rst

```diff
@@ -1,19 +1,150 @@
 Contributing to resqpy
 ======================
 
-As an open-source project, contributions of many forms are most welcome!
+Resqpy is an open source project released under the MIT license. Contributions
+of all forms are most welcome!
 
 How Can I Contribute?
 ---------------------
 
-* Submitting bug reports and feature requests (using the `GitHub issue tracker <https://github.com/bp/resqpy/issues>`_ )
-* Contributing code (in the form of `Pull requests <https://github.com/bp/resqpy/pulls>`_)
-* Documentation or test improvements
-* Publicity and support
+* Submitting bug reports and feature requests (using the `GitHub issue tracker
+<https://github.com/bp/resqpy/issues>`_ ) * Contributing code (in the form of
+`Pull requests <https://github.com/bp/resqpy/pulls>`_) * Documentation or test
+improvements * Publicity and support
+
+Contributors
+------------
+
+Resqpy was created by Andy Beer.
+
+All contributors (alphabetical order):
+
+* Andy Beer * Connor Tann * Emma Nesbit * Nathan Lane
```

Comment:
One other person wrote a small section of code – I need to to check his name! (Casey?)